Source/Core/Core/FifoPlayer/FifoPlayer.h

```diff
@@ -139,6 +139,8 @@ class FifoPlayer
 
   bool IsRunningWithFakeVideoInterfaceUpdates() const;
 
+  Core::System& GetSystem() const { return m_system; }
```

Comment:
Perhaps it would be better to give the CPUCore instance friend access, so that this doesn't need to be added to the public interface.

Reply:
Mhm, yeah makes sense.